stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/client/docusaurus/Push.kt

```diff
@@ -87,6 +88,8 @@ class Push {
                 context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
             }
 
+            override fun onNotificationPermissionStatus(status: NotificationPermissionStatus) { /* no-op */ }
```

Comment:
We should also updated the snippet in docs: https://getstream.io/chat/docs/sdk/android/client/guides/push-notifications/#customize-notification-style